src/contentScript/selectionTooltip/tooltipController.tsx

```diff
@@ -87,6 +95,7 @@ function createTooltip(): HTMLElement {
   container.append(popover);
 
   selectionTooltip = popover;
+
```

Comment:
> Just setting "popover" attribute doesn't promote the element to the top layer

Your comment is likely correct. `popover` hides the the element and then you can call showPopover, probably here, without setting the z-index

```suggestion
  popover.showPopover();
```

Reply:
Other APIs like `dialog` also have similar behaviors/limitations by the way, so we call `.showModal()` on creation:

https://github.com/pixiebrix/pixiebrix-extension/blob/1e6b1a98d0707d76a2e99daffc50540f2d5976f5/src/contentScript/modalDom.tsx#L67-L76

Reply:
I tried calling showPopover (first setting the `display: none`) but didn't get it working immediately. Going to just set z-index for now given that I think there will be additional work required to get positioning to work properly